src/utils/getMarkSize.js

```diff
@@ -0,0 +1,77 @@
+import { SWITCHERS_PARAMS, LEGEND_MARKS_SIZES } from '../constants';
+
+export default function getMarkSize(value, switchersState) {
+  const { ABSOLUTE_CASES } = SWITCHERS_PARAMS.PART_OF_POPULATION;
+  const { CASES_PER_HUNDRED } = SWITCHERS_PARAMS.PART_OF_POPULATION;
+  const { ALL_TIME } = SWITCHERS_PARAMS.TYPE_OF_TIME;
+  const { LAST_DAY } = SWITCHERS_PARAMS.TYPE_OF_TIME;
+  const { CONFIRMED } = SWITCHERS_PARAMS.STAGE_OF_DISEASE;
+  const { DEATHS } = SWITCHERS_PARAMS.STAGE_OF_DISEASE;
+  const { RECOVERED } = SWITCHERS_PARAMS.STAGE_OF_DISEASE;
+
+  const { partOfPopulation, typeOfTime, stageOfDisease } = switchersState;
+
+  const isAbsoluteCases = partOfPopulation === ABSOLUTE_CASES;
+  const isCasesPerHundred = partOfPopulation === CASES_PER_HUNDRED;
+  const isAllTime = typeOfTime === ALL_TIME;
+  const isLastDay = typeOfTime === LAST_DAY;
+  const isConfirmed = stageOfDisease === CONFIRMED;
+  const isDeaths = stageOfDisease === DEATHS;
+  const isRecovered = stageOfDisease === RECOVERED;
+
+  if (isConfirmed || isRecovered) {
+    if (isAbsoluteCases && isAllTime) {
+      if (value > 5000000) return LEGEND_MARKS_SIZES.xl;
```

Comment:
'magic numbers' here and below

Reply:
https://github.com/SorokinTim/covid-dashboard-rss/pull/20/commits/ff66fc42e3a22e9972ce051a57f2cb596eb5ba74